dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/merger/MergerFactory.java

```
@@ -49,6 +53,11 @@
                 result = mergerCache.get(returnType);
             }
         }
+
+        if (result == null) {
```

I don't think we should throw an exception, let's just return null.

---

May be we can add some comments about this NullPointerException

---

I still think it is ok to return null.